docs/requirements.txt

```diff
@@ -7,6 +7,7 @@ ipykernel
 nbsphinx
 recommonmark
 sphinx_markdown_tables
+ipython_genutils
```

Comment:
how is this related?

Reply:
It is not, our doc builds were failing so I added it here but I've already merged it in another PR. I'll remove it here.